JS/ViewControllers/libraryChooseMediaLanguageViewController.js

```diff
@@ -0,0 +1,118 @@
+                <p id="demo"></p>
+                <div class="row row-50">
+                    <div class="col s12 center">
+                        <a id="choose-button" class="waves-effect waves-light btn-large disabled">choose</a>
+                    </div>
+                </div>
+                <div class="row row-100">
+                    <div class="col s12">
+                        <a class="btn-small right" href="index.html">Back to my menu</a>
+                    </div>
+                </div>
+            </section>
+        `;
+        super.renderHtml(htmlView);
+      }
+
+      setupProperties() {
+        this.languagePicker = $('#select-language');
+        this.chooseButton = $('#choose-button');
+        this.getAvailableLanguage();
+      }
+
+      setupEventListeners() {
+        this.languageValueChanged = this.languageValueChanged.bind(this);
+        this.chooseButtonClicked = this.chooseButtonClicked.bind(this);
+
+        this.languagePicker.change(this.languageValueChanged);
+        this.chooseButton.on('click', this.chooseButtonClicked);
+      }
+
+      presentNextController() {
+        const librarySelectMediaLanguageViewController = new LibrarySelectMediaLanguageViewController();
+        librarySelectMediaLanguageViewController.language = this.language;
+        librarySelectMediaLanguageViewController.shortLanguage = this.shortLanguage;
+        this.navigationController.present(librarySelectMediaLanguageViewController);
+      }
+
+      languageValueChanged(){
+        var e = document.getElementById("select-language");
+        this.language = e.options[e.selectedIndex].text;
+        this.shortLanguage = e.options[e.selectedIndex].value;
+        this.setupChooseButton();
+      }
+
+      setupChooseButton() {
+          const shouldEnableSelectButton = this.language !== "";
+          if (shouldEnableSelectButton) {
+              this.chooseButton.removeClass("disabled");
+          } else {
+              this.chooseButton.addClass("disabled");
+          }
+      }
+
+      chooseButtonClicked() {
+          this.presentNextController();
+      }
+
+      // Private Methods
+
+      // get available labguage from online library
+      getAvailableLanguage(){
+        var tmp = "Null";
+          $.ajax({
+             type: 'POST',
+             url: "http://www.st.fmph.uniba.sk/~hrebenar3/Projects/MBP/api.php",
+             crossOrigin: true,
+             data:{
+                 action: "get-avail-lang"
+             },
+             xhrFields: {
+               withCredentials: false
+             },
+             headers:{
+               "x-requested-with": true
+             },
+             dataType: 'json',
+             async: false,
+             success:function(data){
+                 tmp = data;
+                 var attrValue = data["data"];
+                 for (var attr in attrValue){
+                     var x = document.getElementById("select-language");
```

Comment:
tu zasa vo for cykle kazdy jeden raz prezeres cely dokument, pricom uz to mas odchytene v `this.languagePicker` a plus by bolo vhodne nepouzivat premmenne s nazvom `x`, co to znamena `x`